backend/src/test/kotlin/org/loculus/backend/controller/submission/GetReleasedDataEndpointTest.kt

```diff
@@ -149,14 +149,8 @@ class GetReleasedDataEndpointTest(
                 "releasedDate" to TextNode(currentDate),
                 "submittedDate" to TextNode(currentDate),
                 "dataUseTermsRestrictedUntil" to NullNode.getInstance(),
-                "booleanColumn" to BooleanNode.TRUE,
             )
 
-            assertThat(
```

Comment:
Why did you remove this test? It still passes for me locally in this branch

Reply:
It's in commit message of https://github.com/loculus-project/loculus/pull/3232/commits/c62496c05e4ee2f4f19717859ce46af5efd1fae3 :) 

Basically why check on size if you check for each field exactly. The latter is much more informative and has the same test strength. Hence removing the weaker, less informative test that cloaks the better information.

Reply:
Ah true! Yes then deleting it sounds good!